crawler/rabbit/messages/create_plate_message.py

```diff
@@ -81,14 +95,23 @@ def __init__(self, body):
         self._body = body
 
         self.validated_samples = 0
-        self._errors = []
+        self._textual_errors = []
+        self._feedback_errors = []
 
         self._duplicated_sample_values = None
         self._samples = None
 
     @property
-    def errors(self):
-        return self._errors.copy()
+    def textual_errors(self):
+        return self._textual_errors.copy()
```

Comment:
This is just best practice.  Don't expose mutable types as properties.  Create copies of mutable types that are public properties so that the person accessing them cannot modify them directly.